File: Web/RMA-Web-Shell/projects/clientcare/src/app/policy-manager/shared/entities/roleplayer-relation.ts
import { RolePlayerRelationLife } from "./roleplayer-relation-life";

export class RolePlayerRelation {
  id: number;
  fromRolePlayerId: number;
  toRolePlayerId: number;
  rolePlayerTypeId: number;
  policyId?: number;
  allocationPercentage?: number = 0;
  rolePlayerRelationLife: RolePlayerRelationLife;
}

File: Service/ClientCare/RMA.Service.ClientCare.Contracts/Entities/RolePlayer/RolePlayerRelation.cs
namespace RMA.Service.ClientCare.Contracts.Entities.RolePlayer
{
    public class RolePlayerRelation
    {
        public int Id { get; set; }
        public int FromRolePlayerId { get; set; }
        public int ToRolePlayerId { get; set; }
        public int RolePlayerTypeId { get; set; }
        public int? PolicyId { get; set; }
        public int? AllocationPercentage { get; set; }
        public RolePlayerRelationLife RolePlayerRelationLife { get; set; }
    }
}
